packages/adminrouter/extra/src/nginx.master.conf

```diff
@@ -368,6 +349,7 @@ http {
                 auth.access_mesosdns_endpoint();
             }
             include common/proxy-headers.conf;
+            proxy_set_header Authorization "";
```

Comment:
Every `.conf` directive adds a layer of indirection and makes it a bit more difficult to understand the config without having multiple VIM/EMACS/<put-your-favourite-editor-here> tabs open. So far I have been using "More than one directive in the .conf file" as a rule of thumb. Also, having 1-line includes feels like renaming NGINX configuration directives. OTOH "naming" things may be helpful when trying to understand the configuration.

I am like 60/40 against it. If you feel strongly about it - I will be happy to discuss it further.